### PR TITLE
PMM-7626 go modules

### PR DESCRIPTION
**Pull request by Dasio**
FB: https://github.com/Percona-Lab/pmm-submodules/pull/1600
gofakeit was updated to /v6 using modules

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@ all: build
 
 init:           ## Installs tools to $GOPATH/bin (which is expected to be in $PATH).
 	curl https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GOPATH)/bin
-	go install ./vendor/github.com/jstemmer/go-junit-report
+	go install github.com/jstemmer/go-junit-report
```
Comment (artemgavrilov) [inline Makefile:5]: Now version of this tool will be "random". I think we should pin it, as we do it in platform repos

Comment (BupycHuk) [reply Makefile:5]: @Dasio ^

File: init.go [hunk 2]
```diff
@@ -111,6 +111,7 @@ func Transport(baseURL *url.URL, insecureTLS bool) *httptransport.Runtime {
 func init() {
 	seed := time.Now().UnixNano()
 	rand.Seed(seed)
+	gofakeit.SetGlobalFaker(gofakeit.NewCustom(NewConcurrentRand(seed)))
```
Comment (artemgavrilov) [inline init.go:114]: `Faker` is already thread-safe, so it can be:

```suggestion
	gofakeit.SetGlobalFaker(gofakeit.New(seed))
```

**Review by artemgavrilov — CHANGES_REQUESTED** [review]

File: init.go (hunk 2 above)
Comment (artemgavrilov) [reply init.go:114]: It also resolves all comments issues  🙂

Comment (Dasio) [reply init.go:114]: doesn't seems so https://github.com/brianvoe/gofakeit/issues/32

**Review by artemgavrilov — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Review by oter — APPROVED** [review]

**Pull request merged**